src/services/invitation/index.ts

```diff
@@ -55,6 +57,38 @@ const plugin: FastifyPluginAsync = async (fastify, options) => {
     },
   );
 
+  fastify.register(fastifyMultipart, {
+    limits: {
+      fields: MAX_NON_FILE_FIELDS, // Max number of non-file fields (Default: Infinity).
+      fileSize: MAX_FILE_SIZE, // For multipart forms, the max file size (Default: Infinity).
+      files: MAX_FILES, // Max number of file fields (Default: Infinity).
+    },
+  });
+
+  fastify.route<{ Querystring: IdParam; Body: any }>({
```

Comment:
If you do not need the body please do not type it as `any` remove it from the generic typing.